finetune/target_models/sequence_labeling.py

```diff
@@ -249,6 +282,18 @@ def finetune(self, Xs, Y=None, context=None, update_hook=None):
 
             # Reinitialize the model including rebuilding the saver.
             self._initialize()
+
+            # If training with partial labels, combine output of auto negative sampling on
+            # gold data with partial labels data
+            if X_partial and Y_partial:
+                Xs = Xs + X_partial
+                Y = Y + Y_partial
+
+                # Set empty chunks ratio to 0 to remove empty chunks from partially labeled
+                # data, which will only have a sample of true labels
+                # TODO Determine if we need something more sophisticated for chunking
```

Comment:
This is the only thing I'm really unsure on

Reply:
I think this is the most reasonable choice.